### ISSUE TITLE
Accessibility: Video import module accessibility

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#527]
**Describe the bug**
This falls under the accessibility issue of "Inline frame without a text alternative." When users upload a video using the video module, the iFrame that is added does not meet WCAG 2.1 AA standards.

**Expected behavior**
Videos added using the video module should have the title attribute added (either automatically or manually by user).

https://my2.siteimprove.com/Accessibility/1081272/NextGen/Issue?ruleId=13&pageSegments=&issueKind=1&tagFilterType=2&conformance=&siteGoal=

**Comment by kevinkidwell** [comment]
@elrayle I've been looking at this all day so far and I'm not sure if I'm understanding where I would need to edit to either grab the title of the video being added or how to add a text input field to allow users to write the title that we could then put in the HTML.

I've found that all of these modules are coming from Sir Trevor but beyond that I'm not figuring out how they're linked or where to make these kinds of edits.

**Comment by elrayle** [comment]
I took a look at this early on myself and also couldn't find an easy way to make the adjustment.  A first step might be to add an issue to [Sir Trevor](https://github.com/madebymany/sir-trevor-rails).  It would be better if it were fixed in their source.
